### ISSUE TITLE
Large capacity additions for no reason

### ISSUE THREAD

**Issue by HauHe** [KTH-dESA/OSeMOSYS_step#47]
Hi @trevorb1,
I notice large amounts of NewCapacity for `IMPHCO`, `IMPURN`, and `RLu` from step 1 and onwards. Always exactly 10 million. But I don't see why the model is doing so.

**Comment by trevorb1** [comment]
Hi @HauHe! This is quite odd. It seems like maybe CBC is introducing this for some reason. 

If you pull out a model run that introduces the 10million value (I used `steps/step_1/1E0-1C0/`) and solve the model using GLPK, the new capacity results seem fine! However, if I solve it using CBC, these values are added in for some reason. 

What solver are you using when you get this issue? CBC as well? 

![image](https://github.com/KTH-dESA/OSeMOSYS_step/assets/67297083/2469b166-6476-4ade-8d32-861907d705f7)

**Comment by HauHe** [comment]
Yes, I'm also using CBC. 
That's a strange bug. If it doesn't occur in glpk... I will test gurobi as well to see what happens there.

**Comment by HauHe** [comment]
So with gurobi I have these installations of 1000000 as well. But with GLPK neither...
Can't really say more than that it is odd...

**Comment by HauHe** [comment]
Ok, this capacity issue is solved when removing CapitalCost of 0 in the data file and setting the default to 0.000001

**Comment by trevorb1** [comment]
Nice catch, @HauHe! So we are good to close this issue?

**Comment by HauHe** [comment]
Yes, even though results still don't match. But this issue is fixed.
